bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -365,6 +365,15 @@ spec:
     - metadata:
         name: data
         labels: {{- include "common.labels.matchLabels" . | nindent 10 }}
+        {{- if or .Values.persistence.annotations  .Values.commonAnnotations }}
```

Comment:
If `commonAnnotations` was set from before (since it was an existing value), it would cause a breaking change for anyone upgrading the chart to the new release.

To avoid that, could you remove it for now? That way we can add `commonAnnotations` in a future iteration of the chart, as part of internal standarization efforts.

Reply:
FYI this is the upgrade error a user would see:

```
Error: UPGRADE FAILED: cannot patch "my-rabbitmq" with kind StatefulSet: StatefulSet.apps "my-rabbitmq" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
```

Reply:
oh, really, I did not consider that such a problem could arise

I removed commonAnnotations from pvc